jsk_pcl_ros_utils/scripts/xyz_to_screenpoint.py

```diff
@@ -26,8 +31,18 @@ def camera_info_cb(self, msg):
     def point_stamped_cb(self, msg):
         if not self.is_camera_arrived:
             return
-        point = (msg.point.x, msg.point.y, msg.point.z)
-        u, v = self.cameramodels.project3dToPixel(point)
+        pose_stamped = PoseStamped()
+        pose_stamped.pose.position.x = msg.point.x
+        pose_stamped.pose.position.y = msg.point.y
+        pose_stamped.pose.position.z = msg.point.z
+        try:
+            transform = self.tf_buffer.lookup_transform(self.frame_id, msg.header.frame_id, rospy.Time(0), rospy.Duration(1.0))
+        except (tf2_ros.LookupException, tf2_ros.ConnectivityException, tf2_ros.ExtrapolationException) as e:
+            print(e)
```

Comment:
it is better to use `rospy.logerr`
```suggestion
            rospy.logerr('lookup_transform failed: {}'.format(e))
```